### ISSUE TITLE
[enhancement] add support for rdma-cni and rdma namespace mode exclusive

### ISSUE THREAD

**Issue by adrianchiris** [k8snetworkplumbingwg/sriov-network-operator#481]
to fully support RDMA namespaces in sriov-network-operator it would entail the following:

configure rdma subsystem to exclusive mode on node boot[1]
deploy rdma-cni[3]
SriovNetwork to support chaning rdma-cni (can be done using meta plugins[4] so no change needed here)
IbSriovNetwork to support rdmaIsolation ib-sriov-cni attribute [5]
for 3. we should probably look into moving rdma-cni to npwg first as now also Intel supports RDMA.

[1] https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/156
[2] https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/393
[3] https://github.com/Mellanox/rdma-cni
[4] https://github.com/k8snetworkplumbingwg/sriov-network-operator#chaining-cni-metaplugins
[5] https://github.com/k8snetworkplumbingwg/ib-sriov-cni#configuration-reference

**Comment by SchSeba** [comment]
Hi @adrianchiris be aware that `chaning rdma-cni` can't be done in sriovNetwork it's too late.
you need to change this mode on boot before the system even load the RDMA driver if not it doesn't work.

**Comment by cyclinder** [comment]
@SchSeba If we want to change the rdma namespace mode, We need to reboot the node, right?

**Comment by SchSeba** [comment]
that is right you need to do it before the driver gets loaded into the system

**Comment by SchSeba** [comment]
This feature is implemented in the sriov-operator now.
Closing
